guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -3,16 +3,48 @@
+Filters, by default, do not resolve any dependencies of the packages listed in the filters.
 This might require some level of testing to determine what dependencies are required.
 
 If you add a filter that excludes some packages that are required and the Content View has dependency resolution enabled, {Project} ignores the rules you create in your filter in favor of resolving the package dependency.
 
-If you create a content filter for security purposes, to resolve a package dependency, {Project} can add packages that you might consider insecure.
+If you create a content filter for security purposes but choose to resolve dependencies, {Project} can add packages that you might consider insecure.
+
+.Filtering and Dependency Solving Recommendations
+
+Be mindful when combining exclusion filters with dependency solving.
+For example, take a scenario where a user wants to recreate RHEL 8.3 via content view filters and include select packages and errata from a later RHEL 8 y-release.
+The user excludes all errata after the RHEL 8.3 release date minus some select few that are needed and enables dependency solving.
+Dependency solving may pull in more RPMs than expected, which would cause the system to stray further from being a RHEL 8.3 machine.
+If dependency solving pulls in too many unwanted RPMs, consider identifying by hand the core basic dependencies that the extra packages and errata need.
+
+If, in the above case, the user does not need the extra errata and RPMs, then content view filtering is not needed at all.
+The RHEL 8.3 repository could be enabled and used directly from the Red Hat Repositories page.
+
+In some cases, excluding RPMs may make dependency solving impossible.
+For example, if a user wants to make a RHEL 8.3 repo with a few RPMs missing, dependency solving should not be used because the missing RPMs could be dependencies for another RPMs in the repository.
+
+If an erratum is being incrementally added to a recently published content view version, dependency solving is likely unneeded.
+
+If the content view version is out-of-date, a newer erratum's packages may rely on packages not within the content view version.
+Before incrementally adding the erratum with dependency solving, consider publishing a new content view version instead.
+Incrementally adding the erratum with dependency solving may pull in more packages than desired.
+
+If a single RPM is incrementally added to a content view version, dependency solving may be needed to install that RPM.
+Keep in mind that the dependency solving recommendations above also apply to incremental update.
+
+If dependency solving must be used, consider using more content views with fewer repositories and combining them together using composite content views.
+Dependency solving time increases significantly with more repositories because every repository in a content view is cross-checked for dependencies.
```

Comment:
```suggestion
Dependency solving time increases significantly with more repositories because every repository in a content view is verified for dependencies.
```